django_object_actions/utils.py

```diff
@@ -14,17 +15,28 @@ class BaseDjangoObjectActions(object):
     """ModelAdmin mixin to add object-tools just like adding admin actions."""
     # list to hold each object action tool
     objectactions = []
+    tools_view_name = None
 
-    def get_tool_urls(self):
+    def get_tool_urls(self, urls):
         """Gets the url patterns that route each tool to a special view."""
         tools = {}
+
+        end = '_change'
+        for u in urls:
```

Comment:
I tried to verify the absence of a url with "_change", but can not offer a test for this situation and CI fell on the test coverage. So I deleted this. If you have an idea of the test, I'll do it.